assets/files/etc/keepalived/keepalived.conf.template

```diff
@@ -45,3 +51,21 @@ vrrp_instance DNS {
         chk_dns
     }
 }
+
+vrrp_instance INGRESS {
+    state BACKUP
+    interface ${INTERFACE}
+    virtual_router_id 53
+    priority 40
+    advert_int 1
+    authentication {
+        auth_type PASS
+        auth_pass cluster_uuid_ingress_vip
+    }
+    virtual_ipaddress {
+        192.168.111.4
```

Comment:
I set the IP hardcoded to 192.168.111.4 as a temporary workaround to enable ocp routing e2e test on CI.